packages/devextreme/js/renovation/ui/editors/check_box/check_box_icon.tsx

```diff
@@ -3,7 +3,8 @@ import {
 } from '@devextreme-generator/declarations';
 import getElementComputedStyle from '../../../utils/get_computed_style';
 import { hasWindow } from '../../../../core/utils/window';
-import { normalizeStyleProp } from '../../../../core/utils/style';
+
+// import { normalizeStyleProp } from '../../../../core/utils/style';
```

Comment:
Don't forget remove excess comments in this file. Also, failed jest tests are in 'jest test' job